Tensile/Common.py

```diff
@@ -2048,7 +2048,10 @@ def GetAsmCaps(isaVersion: IsaVersion, compilerVersion: CompilerVersion) -> Dict
     derivedAsmCaps["HasLshlOr"]             = tryAssembler(isaVersion, "v_lshl_or_b32 v47, v36, 0x2, v34")
     derivedAsmCaps["HasSMulHi"]             = tryAssembler(isaVersion, "s_mul_hi_u32 s47, s36, s34")
 
-    derivedAsmCaps["HasWMMA"]               = tryAssembler(isaVersion, "v_wmma_f32_16x16x16_f16 v[0:7], v[8:15], v[16:23], v[0:7]")
+    useWavefront64 = (isaVersion[0] >= 10)
+    inst = "v_wmma_f32_16x16x16_f16 v[0:3], v[8:15], v[16:23], v[0:3]" if useWavefront64 else "v_wmma_f32_16x16x16_f16 v[0:7], v[8:15], v[16:23], v[0:7]"
+    derivedAsmCaps["HasWMMA"]               = tryAssembler(isaVersion, inst)
+
```

Comment:
we're either setting this flag or not for the tests, we should pick one or the other, instead of a redundant conditional